lib/src/usage.dart

```diff
@@ -261,3 +310,20 @@ String padRight(String source, int length) {
 
   return result.toString();
 }
+
+bool isWhitespace(String text, int index) {
+  final int rune = text.codeUnitAt(index);
+  return ((rune >= 0x0009 && rune <= 0x000D) ||
```

Comment:
Nit: None of these parens are needed. If you prefer the ones around the `&&`, that's OK. But the outermost ones are definitely unnecessary.

Reply:
Removed the parens.